src/core/ugen_stk.h

```diff
@@ -821,10 +822,10 @@ class DelayL : public Delay
   /*!
     This method is valid only for delay settings greater than zero!
    */
-  MY_FLOAT nextOut(void);
+  virtual MY_FLOAT nextOut(void);
 
```

Comment:
this class of warnings came from implicit virtual function shadowing or overriding, the fix seems to be making overridden methods virtual and adding `using` declarations before overloaded methods